site/docs/v1/tech/identity-providers/samlv2/azure-ad.adoc

```diff
@@ -0,0 +1,179 @@
+DzANBgNVBAgMBkRlbnZlcjETMBEGA1UECgwKRnVzaW9uQXV0aDEWMBQGA1UEAwwN
+RnVzaW9uQXV0aC5pbzETMBEGA1UECwwKRnVzaW9uQXV0aDEkMCIGCSqGSIb3DQEJ
+ARYVcmljaGFyZEBwaWVkcGlwZXIuY29tMB4XDTIyMDcxNjA1MDc1N1oXDTIzMDcx
+NjA1MDc1N1owgYYxCzAJBgNVBAYTAnVzMQ8wDQYDVQQIDAZEZW52ZXIxEzARBgNV
+BAoMCkZ1c2lvbkF1dGgxFjAUBgNVBAMMDUZ1c2lvbkF1dGguaW8xEzARBgNVBAsM
+CkZ1c2lvbkF1dGgxJDAiBgkqhkiG9w0BCQEWFXJpY2hhcmRAcGllZHBpcGVyLmNv
+bTCBnzANBgkqhkiG9w0BAQEFAAOBjQAwgYkCgYEA2FmZJHJFVEM9JwFd6Za87T0Z
+MtIL5djSFC/TBVqhCx15eauNGAV/RoulESA6qsI4LNrbJ8uEYDQa9UXAZCc9yRMa
+e/+E5XAApV4K06duo+vKon5L21YZ7HxzjUfL3bhLqKvpFWCQkNrH0rxgPCGwzh7N
+T24sFcKwaVvBdknm9i8CAwEAAaNQME4wHQYDVR0OBBYEFLdhYvqAwTBCEQsSZdKj
+UVR57CwDMB8GA1UdIwQYMBaAFLdhYvqAwTBCEQsSZdKjUVR57CwDMAwGA1UdEwQF
+MAMBAf8wDQYJKoZIhvcNAQENBQADgYEAIpQags/uHj0dyFcCtRcPeVEDUqBPZaGO
+M9kbFiGATlMNX4OiDvMUKs7Ag9109o0iLWPvLBQrMDn87fSy6+MUXZRS4asjzJCp
+5aVWSevI85H6xS8WXxFr21etaqfiE88Lw86gK5O4iKtMBtCnWA5iUc2EJ0citQ0G
+Pk8ybmMP1r8=
+-----END CERTIFICATE-----
+```
+
+== Create a SAML v2 Identity Provider
+
+Returning to the FusionAuth Admin UI, navigate to `Settings > Identity Providers > Add`.
+
+The identity provider we will be adding is a `SAML V2` identity provider.
+
+image:identity-providers/azure-ad-saml/idp-saml-configuration[IdP SAML configuration,width=1200,role=shadowed]
+
+The following fields will need to be completed.
+
+- _Name_. This is user-defined.
+- _Idp endpoint_. This value can be obtained from your AzureAD Application as demonstrated below.
+
+image:identity-providers/azure-ad-saml/step-four-idp-endpoint[Step Four,width=1200,role=shadowed]
+
+- _Verification key_. For this value, please use the dummy certificate imported above. We will change this later.
+
+Be sure to select `Debug Enabled`. This will ensure debug information is written to the `Event Log`. Remember to disable this option in production.
+
+[NOTE]
+====
+More information regarding the event log can be found below
+
+- link:https://site-local.fusionauth.io/docs/v1/tech/apis/event-logs#overview[API for Event Log]
+- link:https://site-local.fusionauth.io/docs/v1/tech/admin-guide/troubleshooting#event-log[Admin UI and Troubleshooting]
+====
+
+Lastly, optionally configure the Relay State and Logout URL (not all integrations will require this).
+
+Ensure you have enabled this IdP in your FusionAuth application.
+
+image:identity-providers/azure-ad-saml/enable-idp[Enable Idp On Application,width=1200,role=shadowed]
+
+Once saved, you may recieve a CORS warning. Be sure to follow the onscreen instructions and this link:/docs/v1/tech/identity-providers/samlv2/#cors-configuration[documentation].
+
+=== Linking Options
+
+To note, if you are link:/docs/v1/tech/identity-providers/#linking-strategies[linking to a user on email], please configure the SAML Idp to find the email claim. This can be done by navigating to `Settings > Identity Providers > Your SAML Idp > Edit > Options Tab` and adjusting the `Email claim` field.
```

Comment:
```suggestion
If you are link:/docs/v1/tech/identity-providers/#linking-strategies[linking to a user on email], please configure the SAML Idp to find the email claim. This can be done by navigating to [breadcrumb]#Settings -> Identity Providers -> Your SAML IdP -> Edit# and then the [breadcrumb]#Options# tab. Adjust the [field]#Email claim# field.
```